v2/src/linea/types.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers } from 'ethers';
+import type { HexString, HexString32 } from '../types.js';
+
+export const ROLLUP_ABI = new ethers.Interface([
```

Comment:
Just cognisant of the fact that this isn't a type. Same applies to other chains. Perhaps another abis.ts file for each chain?

Reply:
i found it useful during development to have the abis outside of the rollup code so i could test/debug while the rollup/gateway code was unfinished

you're correct they aren't types, but they're type-adjacent

this is something that viem port would probably address